taskt/Core/Automation/Commands/Data Commands/FormatDataCommand.cs

```diff
@@ -1,77 +1,74 @@
+    [Group("Data Commands")]
+    [Description("This command converts a text (either a date or a number) to a specified format and saves result in a variable.")]
     public class FormatDataCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please supply the value or variable (ex. [DateTime.Now]")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Specify either text or a variable that contains a date or number requiring formatting")]
-        [Attributes.PropertyAttributes.SampleUsage("[DateTime.Now], 1/1/2000, 2500")]
-        [Attributes.PropertyAttributes.Remarks("You can use known text or variables.")]
-        public string v_InputValue { get; set; }
+        [PropertyDescription("Input Data")]
+        [InputSpecification("Specify either text or a variable that contains a date or number requiring formatting.")]
+        [SampleUsage("1/1/2000 || 2500 || {DateTime.Now} || {vNumber}")]
+        [Remarks("You can use known text or variables.")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        public string v_InputData { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please select the type of data")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Date")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Number")]
-        [Attributes.PropertyAttributes.InputSpecification("Indicate the source type")]
-        [Attributes.PropertyAttributes.SampleUsage("Choose **Date** or **Number**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Input Data Type")]
+        [PropertyUISelectionOption("Date")]
+        [PropertyUISelectionOption("Number")]
+        [InputSpecification("Select the type of input data.")]
+        [SampleUsage("")]
+        [Remarks("Select 'Date' if input data is Date or 'Number' if it is a Number. Input data of other types will cause error.")]
         public string v_FormatType { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Specify required output format")]
-        [Attributes.PropertyAttributes.InputSpecification("Specify if a specific string format is required.")]
-        [Attributes.PropertyAttributes.SampleUsage("MM/dd/yy, hh:mm, C2, D2, etc.")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Output Data Format")]
+        [InputSpecification("Specify the output data format.")]
+        [SampleUsage("MM/dd/yy, hh:mm:ss || C2 || D2 || {vDataFormat}")]
+        [Remarks("You should specify valid format for input data else it will generate error. 'C2' and 'D2' are Number Formats.")]
```

Comment:
Change to "You should specify a valid input data format; invalid formats will result in an error."

Reply:
done